Rakefile

```diff
@@ -22,74 +22,51 @@ task :generate_host_config => [:check_acceptance]do |t, args|
   sh "cat acceptance/hosts.cfg"
 end
 
-desc 'Run acceptance tests for Rototiller'
-task :acceptance => [:check_acceptance, :generate_host_config]do |t, args|
-
-  config = ENV["BEAKER_CONFIG"]
-
-  preserve_hosts = ENV["BEAKER_PRESERVEHOSTS"]
-  keyfile = ENV["BEAKER_KEYFILE"]
-  load_path = ENV["BEAKER_LOADPATH"]
-  pre_suite = ENV["BEAKER_PRESUITE"]
-  test_suite = ENV["BEAKER_TESTSUITE"]
-
-  beaker = "bundle exec beaker "
-  beaker += " --debug"
-  beaker += " --preserve-hosts #{preserve_hosts}" if preserve_hosts != ''
-  beaker += " --hosts #{config}" if config != ''
-  beaker += " --keyfile #{keyfile}" if keyfile != ''
-  beaker += " --load-path #{load_path}" if load_path != ''
-  beaker += " --pre-suite #{pre_suite}" if pre_suite != ''
-  beaker += " --tests #{test_suite}" if test_suite != ''
-  sh beaker
-end
-
-Rototiller::Task::RototillerTask.define_task :check_acceptance do |t|
+Rototiller::Task::RototillerTask.define_task :acceptance => [:generate_host_config] do |t|
   # with a hash
   t.add_env({:name => 'TEST_TARGET',:default => 'centos7-64', :message => 'The argument to pass to beaker-hostgenerator'})
 
   # with new block syntax
-  t.add_env do |env|
-    env.name = 'BEAKER_CONFIG'
+  t.add_flag do |env|
+    env.name = '--hosts'
     env.default = 'acceptance/hosts.cfg'
     env.message = 'The configuration file that Beaker will use'
+    env.override_env = 'BEAKER_CONFIG'
   end
-  t.add_env do |env|
-    env.name = 'BEAKER_PRESERVEHOSTS'
+  t.add_flag do |env|
+    env.name = '--preserve-hosts'
     env.default = 'never'
     env.message = 'The beaker setting to preserve a provisioned host'
+    env.override_env = 'BEAKER_PRESERVEHOSTS'
```

Comment:
i think it might be good to have same spelling of envs and flags e.g.: `BEAKER_PRESERVE-HOSTS`